pkgs/development/compilers/blueprint/default.nix

```diff
@@ -29,18 +30,32 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     python3
-  ];
+    gtk4
+  ] ++ (with python3.pkgs; [
+    pygobject3
+    wrapPython
+  ]);
 
   propagatedBuildInputs = [
-    # So that the compiler can find GIR and .ui files
```

Comment:
Maybe change it to ”For setup hook, so that the compiler can find typelib files”